workflows/argo/chgres-cube.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: chgres-cube
+spec:
+  arguments:
+    parameters:
+    - name: restarts
+    - name: date
+    - name: source_resolution
+      value: C48
+    - name: target_resolution
+      value: C384
+    - name: tracers
+      value: '"sphum","liq_wat","o3mr","ice_wat","rainwat","snowwat","graupel","sgs_tke"'
+    - name: vcoord_file
+    - name: reference_data
+      value: gs://vcm-ml-raw-flexible-retention/2023-02-24-chgres-cube-forcing-data/2023-02-24-chgres-cube-forcing-data
```

Comment:
This is a bit of an unfortunate path.  The duplicate directory name is real.  We could change it, but it would require temporarily lifting the retention policy on the `vcm-ml-raw-flexible-retention` bucket.